modules/va_notify/app/sidekiq/va_notify/icn_job.rb

```diff
@@ -26,6 +27,7 @@ def perform(icn, template_id, personalisation = nil, api_key = Settings.vanotify
           template_id:, personalisation:
         }.compact
       )
+      StatsD.increment('api.vanotify.icn_job.success')
```

Comment:
Probably a nit, but I know we're trying to avoid doing inline disables against the current set of Rubocop rules. Could you maybe bust out a `handle_backend_exception` method or something?

Reply:
Thanks for the review @LindseySaari. Updated to pull out exception handling into it's own method.